components/eam/cime_config/testdefs/testmods_dirs/eam/mmf_variance_transport/shell_commands

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+./xmlchange --append CAM_CONFIG_OPTS="-use_MMF_VT"
+if [ `./xmlquery --value MACH` == chrysalis ]; then ./xmlchange FORCE_BUILD_SMP=TRUE; fi
```

Comment:
We should probably delete this line too.

Reply:
Moved this test mod, will be covered in a separate PR.